docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
+1. [ ]  User testing on mockups (design)
+1. [ ]  go/nogo on individual features (engineering)
+1. [ ]  create release tag for the selected features
+1. [ ]  all automated tests pass
+1. [ ]  all internationalization tests pass
+1. [ ]  generate release notes
+
+## Security checks
+TBD: Andrew will send a list
+
+## Collect store materials
+
+TODO: Maintain full list of assets required in template
+1. [ ] generate iOS text assets
+1. [ ] generate android text assets
+1. [ ] generate iOS screenshots
+1. [ ] generate android screenshots
+
+## Final WHO sign off/approval
+??? not sure what the process for this is
+
+## Push to the stores for beta testing
+
+1. [ ] publish .apk and .ipa to github using automated build
+1. [ ] WHO will fill in store information and upload apk/ipa
```

Comment:
Store information (copy, screenshots) should also be checked into our repo and provided in a standardized or automated form.

Everything that defines a release must be purely reproducible from the release tag.

Reply:
Fair enough. Will fix.